### DISCUSSION TITLE
Enabling select all in Multi-Select dropdown

### DISCUSSION THREAD

**Discussion by EdwinThomas11** [serenity-is/Serenity#7098] [Help]
Hello Team,

I'm seeking a way for users to select all dropdown values at once, rather than individually in the multi-select dropdown. Your assistance on this matter is much appreciated.

Thanks,
Edwin Thomas D

**Comment by LSNicholls** [comment]
Hi Edwin,

This may not work for you but here is what I did, similar to the same requirement in other environments:

1. add a dropdown value such as "*" to the top of the displayed list
2. change the List handler to recognize (request.ContainsText) the special value.

Come to think of it, I haven't done exactly this in Serenity, sorry, it was a bit more specialized but this is what I would try.
